docs/kubernetes-shortguide-definitions/helm-tiller-definition-shortguide/index.md

```diff
@@ -15,4 +15,4 @@ show_on_rss_feed: false
 
 ## Helm Tiller
 
-A server component that runs on your cluster and receives commands from the Helm client software. [Tiller](https://helm.sh/docs/glossary/#tiller) is responsible for directly interacting with the Kubernetes API (which the client software does not do). Tiller maintains the state for your Helm releases.
+A server component that runs on your cluster and receives commands from the Helm client software before it's deprecation in [Helm 3](https://helm.sh/docs/faq/#removal-of-tiller). [Tiller](https://v2.helm.sh/docs/glossary/#tiller) is responsible for directly interacting with the Kubernetes API (which the client software does not do). Tiller maintains the state for your Helm releases.
```

Comment:
This reads a little weird. Maybe just a second sentence that states: "Tiller has been deprecated in Helm 3."

Reply:
added sentence